fdbserver/storageserver.actor.cpp

```diff
@@ -3444,23 +3444,39 @@ ACTOR Future<GetRangeReqAndResultRef> quickGetKeyValues(
 			getRange.result = RangeResultRef(reply.data, reply.more);
 			return getRange;
 		}
+		TraceEvent("QuickGetKeyValuesErrorInReply")
+		    .error(reply.error.get())
+		    .detail("ServerID", data->thisServerID)
```

Comment:
The event can be associated with ID in a more concise way:
`TraceEvent("QuickGetKeyValuesErrorInReply", data->thisServerID)`